docs/quick_run.md

```diff
@@ -264,6 +264,29 @@ python tools/utils/translation_eval.py ./configs/pix2pix/pix2pix_vanilla_unet_bn
     https://download.openmmlab.com/mmgen/pix2pix/pix2pix_vanilla_unet_bn_1x1_80k_facades.py_20210410_174537-36d956f1.pth \
     --eval IS
 ```
+To be noted that, the selection of Inception V3 and image resize method can heavily influence the final result of IS.  Therefore, we strongly recommend users may download the [Tero's script model of Inception V3](https://nvlabs-fi-cdn.nvidia.com/stylegan2-ada-pytorch/pretrained/metrics/inception-2015-12-05.pt) (load this script model need torch >= 1.6) and use `Bicubic` interpolation with `Pillow` backend. We provide a template for the [data process pipline](https://github.com/open-mmlab/mmgeneration/tree/master/configs/_base_/datasets/Inception_Score.py) as well.
```

Comment:
1. `heavily` --> `significantly`
2. the final results of IS ---> the final IS score